.gitignore

```diff
@@ -9,3 +9,14 @@ ci/test/output/
 .env
 *.crt
 *.key
+
+# Build artifacts
+build/
+dist/
+*.egg-info/
+
+# Virtual environment
+venv/
+
+# Misc
+.DS_Store
```

Comment:
I think we can leave this repo's .gitignore alone. It looks like some this stuff crept in from the CLI's gitignore file.